### PR TITLE
ePBFT

### PR DESCRIPTION
**Pull request by ashamis**

### CODE REVIEW

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

File: .azure-pipelines-no-sgx.yml [hunk 1]
```diff
@@ -64,11 +64,16 @@ jobs:
...
     - script: python3.7 notice-check.py
-      displayName: 'Check copyright notices'
+      displayName: 'Check CCF copyright notices'
 
-    - script: ./check-format.sh src samples
+    - script: python3.7 notice-check.py
+      workingDirectory: ePBFT
+      displayName: 'Check ePBFT copyright notices'
+
+    - script: ./check-format.sh src samples ePBFT/src
```
Comment (achamayou) [inline .azure-pipelines-no-sgx.yml]: I think we want ePBFT under src etc, but this is fine to begin with.

File: ePBFT/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,304 @@
+cmake_minimum_required(VERSION 3.10)
```
Comment (achamayou) [inline ePBFT/CMakeLists.txt]: Unless I'm missing something, this isn't called from the CI?

File: ePBFT/sphinx/source/conf.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
+# Copyright (c) Microsoft Corporation.
```
Comment (achamayou) [inline ePBFT/sphinx/source/conf.py]: As discussed, we want to merge this too.

File: src/ds/files.h [hunk 4]
```diff
@@ -2,27 +2,16 @@
 // Licensed under the Apache 2.0 License.
 #pragma once
```
Comment (achamayou) [inline src/ds/files.h]: Thanks for killing the unused stuff!

**Comment by cimetrics** [comment]
images

**Pull request merged**